tests/Microsoft.Identity.Test.Unit/BrokerTests/WamTests.cs

```diff
@@ -1,1575 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.

```

Comment:
Since we still use old WAM on UWP, any way to keep these tests?

Reply:
We do not work on it unless there is Sev1. It has not happened for a long time. So no point in keeping them for UWP.